test/mpi/dtpools/src/dtpools_internal.c

```diff
@@ -13,8 +13,9 @@
 #define TYPE_NAME_MAXLEN (256)
 #define BASIC_TYPE_NAME_MAXLEN (64)
 
-#define DTPI_OBJ_INIT_BUF(c_type, type_ptr)                                  \
+#define DTPI_OBJ_INIT_BUF(par, c_type, type_ptr)                             \
```

Comment:
I may be wrong, but are these changes (that move `i, j, k` into macros and add `par` as a parameter) something that should be in a different commit?

Reply:
That is part of the refactoring. If you prefer I can put them in a different commit.

Reply:
I guess there are two solutions.
1. put this part in a different commit
2. update the commit message to add an explanation on this change

I would prefer 1., unless this part of change is strongly related with other changes (e.g. introducing function pointers for destruction/checking), to reduce the size of each patch.

Reply:
> test/dtpools: minor cleanups

The commit message needs to be improved. I think it should describe a summary of changes and why it is needed.